src/main/resources/db/migration/postgresql/V7.0__add_public_private_posts.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE negotiation RENAME COLUMN posts_enabled TO public_posts_enabled;
+
+ALTER TABLE negotiation ADD COLUMN private_posts_enabled boolean;
+
+UPDATE negotiation n
+SET public_posts_enabled = true
+WHERE n.current_state in ('SUBMITTED', 'APPROVED', 'DECLINED', 'IN_PROGRESS', 'PAUSED', 'CONCLUDED');
```

Comment:
just In progress is enough

Reply:
at least SUBMITTED. It's the reason why we're doing this change :)